gh-98776: Fix make regen-test-levenshtein for out-of-tree builds (#98779)
python/cpython: Misc/NEWS.d/next/Build/2022-10-27-19-47-31.gh-issue-98776.lt_UOG.rst

> When building Pyton out-of-tree, don't crash during ``make

Reviewer: ```suggestion
When building Python out-of-tree, don't crash during ``make
```

> When building Python out-of-tree, don't crash during ``make
> regen-test-levenshtein``.

Reviewer 2: ```suggestion
Fix ``make regen-test-levenshtein`` for out-of-tree builds.
```
